docs/source/history.rst

```diff
@@ -0,0 +1,43 @@
+History
+=======
+
+The history and versioning information is exposed using the @history endpoint.
+It lists the historical versions of the content.
+
+See :ref:`content_get_version` for documentation on reading older versions of a resource.
+
+
+GET Resource history
+--------------------
+
+Listing versions and history of a resource:
+
+..  http:example:: curl httpie python-requests
+    :request: _json/history_get.req
+
+.. literalinclude:: _json/history_get.resp
+   :language: http
+
+This following fields are returned:
+
+-  action: the workflow transition id, or 'Edited' for versioning.
+- actor: the user who performed the action. This contains a subobject with the details.
+- comments: a changenote
+- content_url: link to the content endpoint of this specific version.
+- may_revert: true if the user has permission to revert.
+- time: when this action occured in ISO format.
+- transition_title: the workflow transition's title, for 'Edited' for versioning.
+- type: 'workflow' for workflow changes, 'versioning' for editing, or null for content creation.
+- version_id: identifier for this specific version of the resource.
+
+
+PATCH revert historical versions
+--------------------------------
+
+Reverting to older versions of a resource can be done by issueing a PATCH request with a version_id.
```

Comment:
@jaroel I did not come to a final conclusion yet. Though, I am not convinced yet that PATCH is the correct HTTP verb. PATCH is a pretty specific HTTP verb to indicate a change on a document by providing a DIFF of the original document. 

One could argue (and this is the reason for your choice I guess), that a version change is just a minor change to the original document. Though, a PUT is usually used to completely replace the document. A PATCH with just the version number will lead to much more on the server than just changing the version_id on the document.

If we would use PUT, we would have to send the entire document though (or at least the link to that document).

There are people that argue the body of a PATCH should include a real diff of the original document only:

http://williamdurand.fr/2014/02/14/please-do-not-patch-like-an-idiot/

In any case, I think we need to make sure that however we use PATCH, we use it in a consistent way. I am just thinking out loud here and I am open for suggestions. cc @buchi @lukasgraf @sneridagh @bloodbare